core/src/com/unciv/ui/worldscreen/unit/UnitActions.kt

```diff
@@ -290,7 +291,7 @@ object UnitActions {
             actionList += UnitAction(UnitActionType.Pillage, action = null)
         else actionList += UnitAction(type = UnitActionType.Pillage) {
             if (!worldScreen.hasOpenPopups()) {
-                val pillageText = "Are you sure you want to pillage this [${unit.currentTile.improvement}]?"
+                val pillageText = if(unit.currentTile.improvement == null) "Are you sure you want to pillage this [${unit.currentTile.roadStatus.name}]?" else "Are you sure you want to pillage this [${unit.currentTile.improvement}]?"
```

Comment:
This is a _very_ long line, maybe:
```kotlin
val tile = unit.getTile()
val pillagedImprovement = if (tile.improvement == null) tile.roadStatus.name else tile.improvement
val pillageText = Are you sure you want to pillage this [$pillagedImprovement]?
```
Maybe inline the `tile` if you find it too verbose